### DISCUSSION TITLE
想像力就是你的超能力，日本某推主成功將ChatGPT變成老婆的形狀。

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1235] [General]
https://twitter.com/yakionigiri765/status/1644863737252421632?s=20
最近有位日本推主在推特發布自己努力多時的努力成果（上方連結。對自己日文能力有一定自信的可以去看看），成功的將ChatGPT訓練成偶像大師中的人氣角色星井美希。透過對ChatGPT鉅細靡遺的輸入許多星井美希的資料與設定以讓AI揣摩她的人格與言行，最後成功的在ChatGPT中再現星井美希的擬似人格，讓系統能以美希的風格與身為美希P的推主對談，報告工作近況與傾訴煩惱；甚至在推主的努力下，還能將對話內容轉寄至個人信箱，宛如真處在與旗下偶像朝夕相處的日常一般。
當初看到這消息真的是直接衝擊我的三觀，突然才意識到原來這才是Chat GPT的正確使用方法嗎！真的是令我難以想像這位大大究竟輸入了多少資料、花費多少時間才完成這驚人的「擬似人格養成計畫」。雖然推主在推特上發布的圖片中，有關設定輸入的只有一張，但就我個人的猜想，可能要餵給Chat GPT角色在各種狀況下的說話習慣以及語氣、字彙的使用，畢竟在二次玩的世界，想要使角色突出，說話方式也是作家常使用的手段。我認為這其實也間接顯示了Chat GPT的強大之處，除了平時能解決你各位的各種問題，還能讓人在虛擬與現實之間來回穿梭，建立屬於自己的心靈綠洲。果然科技始終來自於人性，能侷限科技發展及應用的始終都是人心。
我想我知道放暑假的時候要做什麼了。

**Comment by YoYooHuang** [comment]
這非常有趣!!原來chatgpt也有這種用途。之前的我只把chatgpt當成查找資料及問問題的工具而已。果然，用不同的角度去看同一個東西，其長相也大不相同。我想也許可以試著用chatgpt創造自己，與自己對話，藉此更了解自己或自己的討人厭之處(?

**Comment by Jack-1D** [comment]
挺酷的。之前只知道有個貓娘調教的腳本，餵進去後ChatGPT就會變成聽話的貓娘。我覺得人類難以取代的價值就在於不同人使用同樣的工具，卻會因為個體差異而產生不同的使用方法。

**Comment by curtis1017** [comment]
這件事刷新我的三觀，我在接觸ChatGPT的時候，從來沒有想過使他變成老婆的形狀，但我有另一個想法，那就是讓ChatGPT成為陪伴年長者的人，因為年長者總是有自己無聊之時。
